clustered/server/src/main/java/org/ehcache/clustered/server/ServerStoreImpl.java

```diff
@@ -29,16 +29,14 @@
 @CommonComponent
 public class ServerStoreImpl implements ServerStore {
 
-  private static final int OFFHEAP_CHAIN_SEGMENTS = 16;
```

Comment:
just reverted to:

  private static final int OFFHEAP_CHAIN_SEGMENTS = 16;

  private final ServerStoreConfiguration storeConfiguration;
  private final PageSource pageSource;
  private final OffHeapServerStore store;

  public ServerStoreImpl(ServerStoreConfiguration storeConfiguration, PageSource pageSource) {
    this.storeConfiguration = storeConfiguration;
    this.pageSource = pageSource;
    this.store = new OffHeapServerStore(pageSource, OFFHEAP_CHAIN_SEGMENTS);
  }